packages/utils/telemetry-utils/api-report/telemetry-utils.api.md

```diff
@@ -433,6 +434,9 @@ export class UsageError extends LoggingError implements IUsageError, IFluidError
 // @internal
 export function validatePrecondition(condition: boolean, message: string, props?: ITelemetryBaseProperties): asserts condition;
 
+// @alpha
+export const wrapConfigProviderWithDefaults: (original: IConfigProviderBase | undefined, defaults: Record<string, ConfigTypes>) => IConfigProviderBase;
```

Comment:
I'm a bit surprised that nothing complained about `ConfigTypes` not being re-exported from this package; otherwise this function is not usable in a type-safe manner by potential consumers, right?